### PR TITLE
Bug 1175045 - [Messages][NGA] Add serviceworkerware to our codebase and update the script for external lib, r=julienw,azasypkin

### PR DESCRIPTION
**Pull request by steveck-chung**

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=4861a9dc44005f3a4d58e13ddaa4f0d47e127ea6

File: apps/sms/tools/create_bridge_lib.sh [hunk 1]
```diff
@@ -2,7 +2,7 @@
```
Comment (azasypkin) [inline apps/sms/tools/create_bridge_lib.sh]: It's time to rename script :) Maybe ```update_libs.sh``` or any better name?

Comment (steveck-chung) [inline apps/sms/tools/create_bridge_lib.sh]: right, I update the content but forgot to update name. 'update_libs.sh' sounds good to me.

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=cc3c5537a5876ccd2eac3bd0dc1edc774d5507d0

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=749b408afb4fc99710a08f33314af4552d78f5de

**Pull request merged**